broker/broker.go

```diff
@@ -155,6 +155,12 @@ func (b *CdnServiceBroker) LastOperation(
 			"domain": route.DomainExternal,
 			"state":  route.State,
 		})
+		if err == models.CNAMEError {
```

Comment:
Is this going to work? We never return `CNAMEError`, but you're relying on the error we get back being equal.

Even if the message is the same, it feels a bit flakey to compare the entire error message - what if AWS change the wording?

Would it be possible to do something like `if err.message.contains('CNAMEAlreadyExists')` ?

Reply:
That was my original approach so I'll go back to that.